unblob/models.py

```diff
@@ -47,6 +47,8 @@ class Chunk:
 
     id: str = attr.field(factory=new_id)
 
+    host_file: Optional[Path] = None
```

Comment:
I would just use `file` what is `host` here?

Reply:
`parent_file` it will be.

Reply:
`file` is the name